src/services/TalkCommentEmailService.php

```diff
@@ -19,10 +19,23 @@ public function __construct($config, $recipients, $talk, $comment)
     public function sendEmail()
     {
         $this->setSubject("New feedback on " . $this->talk['talk_title']);
+
+        $byLine = '';
+
+        if(isset($this->comment['user_display_name'])) {
+            $byLine = ' by ' . $this->comment['user_display_name'];
+
+            if(isset($this->comment['username'])) {
```

Comment:
Not something I know, so the code is just protective unless I'm told I can remove the second check.